app/cdap/text/text-en.yaml

```diff
@@ -3413,4 +3413,11 @@ features:
     gridTable:
       title: "No rows left in this sample"
       subtitle: "Remove some recipe steps to view some data"
+  Breadcrumb:
+    labels:
+      connectionsList: 'Data Sources'
+      wrangleHome: 'Home'
+    params: 
+      wrangleHome: 'home'
```

Comment:
plz rearrange the labels in alphabetical order

Reply:
done